src/App.tsx

```diff
@@ -2,13 +2,26 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import getPosts from './api/posts';
+import getComments from './api/comments';
+import getUsers from './api/users';
```

Comment:
```suggestion
import posts from './api/posts';
import comments from './api/comments';
import users from './api/users';
```
коли в нас щось починається з get - це означає, що ми так називаємо функцію чи метод, який ДІСТАЄ щось звідкись. Тут же в тебе просто змінні (а не функції), в які записуються значення